src/modules/renderTasks.js

```diff
@@ -0,0 +1,71 @@
+import itemsArray from './itemsArray.js';
+
+const toDoItemsEl = document.querySelector('.to-do-items');
+const editTaskBtnEl = document.getElementsByClassName('editTaskBtn');
+const editTaskEl = document.getElementsByClassName('to-do-edit');
+const toDoTask = document.getElementsByClassName('to-do-task');
+
+const renderToDoItems = () => {
+  let markup = '';
+
+  itemsArray.forEach((elem, index) => {
+    markup += `<section class="to-do-item  border-bottom" id=${index}>
+    <div class="to-do-edit hide flex-row border-bottom">
+    <input type="text" class="to-do-input" placeholder="Input a task or use the delete button...">
+    <span class="material-symbols-outlined to-do-addTask">add-task</span>
+    </div>
+    <div class="flex-row to-do-task">
+    <input type="checkbox" ${elem.completed && 'checked'}>
+    <p class='description'>${elem.description}</p>
+    <span class="to-do-item-icons">
+    <span class="material-symbols-outlined dltTaskBtn" id=${index}>delete</span>
+    <span class="material-symbols-outlined editTaskBtn" id=${index}>edit_note</span>
+    </span>
+    </div>
+  </section>`;
+  });
+
+  const removeTask = () => {
+    const removeBtnsEl = [...document.getElementsByClassName('dltTaskBtn')];
+    removeBtnsEl.forEach((item) => {
+      item.addEventListener('click', (e) => {
+        itemsArray.splice(e.target.id, 1);
+        localStorage.setItem('itemsArray', JSON.stringify(itemsArray));
+        renderToDoItems();
+      });
+    });
+  };
```

Comment:
- [ ] Please make the delete task update all remaining items' indexes to achieve this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Deleting%20a%20task%20should%20update%20all%20remaining%20items%27%20indexes%2C%20so%20they%20represent%20the%20current%20list%20order%20and%20are%20unique(i.e.%20if%20you%27re%20deleting%20the%20first%20task%20index%201%20from%20the%20list%2C%20the%20index%20of%20the%20next%20task(2)%20should%20set%20to%201)..).

Reply:
@mckent05 thanks for your comments, I don't understand what you mean please 
https://www.webmobilefirst.com/en/screencasts/85OS_II7fL/